scripts/android-emulator-images-update.sh

```diff
@@ -2,46 +2,67 @@
 
 # This script will update
 # Intel HAXM (required for accelerated emulators)
-# Android emulator images from 4.2 (android-17) to 7.1 (android-25)
+# Android emulator images from 4.2 (android-17) to 8.1 (android-27)
 #
 # Notes:
 # This script will work on both Linux and macOS hosts!
 # This script will install both arm and x86 emulators plus google api images!
 # Emulator images require a lot of disk space!
 
-echo "Update Intel HAXM"
-echo y | $ANDROID_HOME/tools/android update sdk --filter extra-intel-Hardware_Accelerated_Execution_Manager --all --no-ui
-sudo $ANDROID_HOME/extras/intel/Hardware_Accelerated_Execution_Manager/silent_install.sh
-
-echo "Update Android Emulator Images"
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86_64-google_apis-25 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-25 --all --no-ui
-
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-android-24 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-armeabi-v7a-android-24 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-arm64-v8a-android-24 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-24 --all --no-ui
+set -e
+set -x
 
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-android-23 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-23 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-armeabi-v7a-android-23 --all --no-ui
-
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-android-22 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-22 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-armeabi-v7a-android-22 --all --no-ui
-
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-android-21 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-21 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-armeabi-v7a-android-21 --all --no-ui
+echo "Update Intel HAXM"
+echo y | "${ANDROID_HOME}/tools/bin/sdkmanager" "extras;intel;Hardware_Accelerated_Execution_Manager"
+sudo sh "${ANDROID_HOME}/extras/intel/Hardware_Accelerated_Execution_Manager/silent_install.sh"
 
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-android-19 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-armeabi-v7a-android-19 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-19 --all --no-ui
+PACKAGES=()
```

Comment:
I would keep only the emulator images that are used in the create script and remove the rest to save disk space.

Reply:
ok